internal/model/payments.go

```diff
@@ -0,0 +1,20 @@
+package Model
+
+import "time"
+
+type Payments struct {
+	Id             string    `gorm:"primary_key; column:id" json:"id"`
+	UserId         string    `gorm:"foreign_key; column:user_id" json:"user_id"`
+	RazorpayLinkId string    `gorm:"column:razorpay_link_id" json:"razorpay_link_id"`
+	RazorpayLink   string    `gorm:"column:razorpay_link" json:"razorpay_link"`
+	Amount         int       `gorm:"column:amount" json:"amount"`
+	Type           string    `gorm:"column:type" json:"type"`
+	CurrentBalance int       `gorm:"column:current_amount" json:"current_amount"`
```

Comment:
is this current_amount or current_balance?

Reply:
cahnges made please check